x-pack/plugins/infra/public/components/logging/log_text_stream/log_entry_field_column.tsx

```diff
@@ -24,10 +24,18 @@ export const LogEntryFieldColumn: React.FunctionComponent<LogEntryFieldColumnPro
   isWrapped,
 }) => {
   const value = useMemo(() => JSON.parse(encodedValue), [encodedValue]);
-
+  const formattedValue = Array.isArray(value) ? (
+    <ul>
+      {value.map((entry, i) => (
+        <li key={`LogEntryFieldColumn-${entry}-${i}`}>{entry}</li>
```

Comment:
I've always had React linters complain to me when using just the index in a `key`, because of the risk of overlapping. I can figure out some other form of entropy though.

Reply:
I've never seen our linter complaining and I don't see why it should here :man_shrugging: